### PR TITLE
Add encryption for backup

### PR DESCRIPTION
**Pull request by Bizyroth**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -59,6 +59,13 @@ ENV FILESYSTEM_PATH=""
 ENV SERVER_LISTEN_ADDRESS=":12000"
 ENV SERVER_ADDRESS="http://127.0.0.1:12000"
 
+# generate couple of key with:
+#  openssl req -x509 -nodes -days 1000000 -newkey rsa:4096 -keyout PATHTOPRIVATEKEY \
+# -out PATHTOPUBLICKEY
+ENV BACKUP_PUBLIC_KEY="/var/backup/backupkey.public"
+ENV BACKUP_PRIVATE_KEY="/var/backup/backupkey.private"
+ENV ENCRYPTION_ENABLE=true
```
Comment (shakapark) [inline Dockerfile]: Can you disable it by default ?

File: backupToAWS.sh [hunk 2]
```diff
@@ -156,6 +156,9 @@ function backupPostgresToBucket() {
 
   PGPASSWORD=$POSTGRES_PASSWD pg_dump -h $POSTGRES_HOST -p $POSTGRES_PORT -U $POSTGRES_USER -d $POSTGRES_DATABASE \
   $FILTER_TABLE $EXCLUDE_TABLE $COMPRESSION 2> dump_error.log | \
+  if [ "$ENCRYPTION_ENABLE" = "true" ]; then
```
Comment (shakapark) [inline backupToAWS.sh]: Can you add encryption option with $ENCRYPTION variable, like $COMPRESSION for all backup and restore ?

**Review by shakapark — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (Bizyroth) [reply Dockerfile]: Done

File: backupToAWS.sh (hunk 2 above)
Comment (Bizyroth) [reply backupToAWS.sh]: Done

Comment (shakapark) [reply backupToAWS.sh]: If you write ENCRYPTION=$( _command_ ): the value of ENCRYPTION will be the result of _command_. It must be the command itself

**Review by shakapark — APPROVED** [review]

**Pull request merged**